src/analyses/variable-sensitivity/pointer_abstract_object.h

```diff
@@ -97,6 +97,15 @@ class pointer_abstract_objectt : public abstract_objectt
 protected:
   CLONE
 
+  virtual bool same_target(abstract_object_pointert other) const
```

Comment:
I am not entirely opposed to adding to the interface for pointers if it is necessary.  However there is a general move towards making the interface of `abstract_objectt` and children more uniform so I wonder if there is another way.